setup.py

```diff
@@ -35,14 +35,10 @@ def run(self):
       url='https://github.com/sdelements/django-security',
       classifiers=[
           'Framework :: Django',
-          'Framework :: Django :: 1.8',
-          'Framework :: Django :: 1.9',
-          'Framework :: Django :: 1.10',
           'Framework :: Django :: 1.11',
+          'Framework :: Django :: 2.2',
```

Comment:
Shall we add 3.0 too since the tests are passing?

Reply:
Sure, added it in.